prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -675,7 +675,6 @@ void rxr_pkt_handle_send_completion(struct rxr_ep *ep, struct fi_cq_data_entry *
 
 	switch (rxr_get_base_hdr(pkt_entry->pkt)->type) {
 	case RXR_HANDSHAKE_PKT:
-		break;
```

Comment:
I think you removed it by mistake.

Reply:
The `break` here is not needed, as the next `case RXR_CTS_PKT` will break anyway. 
It's not necessary to open another PR fixing this, so remove it here.

Reply:
Rely on fall back in switch case is not good practice. What if in the future we add some special handling for CTS packet? say call function rxr_pkt_handle_cts_send_completion()?

Reply:
Alright, I will change it back.

Reply:
Fixed in new rev.